src/hashstore/filehashstore.py

```diff
@@ -770,80 +815,130 @@ def retrieve_metadata(self, pid, format_id=None):
         )
         return metadata_stream
 
-    def delete_object(self, pid):
+    def delete_object(self, ab_id, id_type=None):
```

Comment:
I think the behavior of this method is inconsistent with [this doc comment](https://github.com/DataONEorg/hashstore/pull/88/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R117), unless I'm mistaken?

Reply:
After working on HashStore-java, there are a few changes that need to be made to `delete_object` (missing locks). I will review this carefully and revise.